modules/proxyconnection/ProxyConnectionPC.js

```diff
@@ -364,11 +364,18 @@ export default class ProxyConnectionPC {
         this._tracks.forEach(track => track.dispose());
         this._tracks = [];
 
-        if (!this._peerConnection) {
-            return;
+        if (this._peerConnection) {
+            this._peerConnection.onTerminated();
         }
 
-        this._peerConnection.onTerminated();
+        if (this._rtc) {
+            this._rtc.removeListener(
+                RTCEvents.REMOTE_TRACK_ADDED,
+                this._onRemoteStream
+            );
+
+            this._rtc.destroy();
```

Comment:
Thanks !

Reply:
Mobile doesn't have this problem. `DEVICE_LIST_CHANGED` and `AUDIO_OUTPUT_DEVICE_CHANGED` event listeners don't get set on mobile because those get set only when `RTCUtils.isDeviceChangeAvailable('output')` is true, which it's not for mobile. `LASTN_ENDPOINT_CHANGED` listener gets cleaned up on its own if `closeBridgeChannel` is called. `DATA_CHANNEL_OPEN` listener gets removed if the data channel does open but could be leak if it never does open. That said, I'll still try to find a place to call destroy to be safe.

Reply:
Added rtc cleanup on conference leave. Seems like rtc is normally an instance of JitsiConference and its usage is not needed after leave (or so I really, really hope). I can try being less aggressive with the cleanup by at least leaving rtc available on the instance but I null'ed it to follow the pattern of the rest of the conference leave logic.